lib/client.ml

```diff
@@ -67,6 +65,14 @@ let init ~state ~msg_handler (address, port) =
   let state = ref state in
   let recv_mutex = Lwt_mutex.create () in
   let state_mutex = Lwt_mutex.create () in
-  let client = ref { address; port; socket; state; recv_mutex; state_mutex } in
+  let client =
+    ref
+      {
+        address = Address.create_address address port;
```

Comment:
Since it's in Address module, we can just call this function `create`. Seems to be a common idiom.

Reply:
Done!